modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -35,6 +35,22 @@ static AppMode strToAppMode(const std::string& mode_str) {
     }
 }
 
+enum class WaitMode {
+    BUSY,
+    SLEEP
+};
+
+static WaitMode strToWaitMode(const std::string& mode_str) {
+    if (mode_str == "sleep") {
+        return WaitMode::SLEEP;
+    } else if (mode_str == "busy") {
+        return WaitMode::BUSY;
+    } else {
+        throw std::logic_error("Unsupported wait mode: " + mode_str +
+                "\nPlease chose between: busy (default) and sleep");
```

Comment:
`choose`. Also why `BUSY` is the default mode? I mean not because it's `0` in the enum

Reply:
Debatable what should be default but `BUSY` works more accurate which is critical for our use cases

Reply:
I mean there are two options for us:
1. Leave `SLEEP` by default and specify `wait_mode: busy` for every source in config
2. Leave `BUSY` by default.

I've decided the second option